rust/shared/proto/ziofa.proto

```diff
@@ -31,7 +31,8 @@ message ProcessList {
 message Process {
     int32 pid = 1;
     int32 ppid = 2;
-    string comm = 3;
+    // if cmdline would be empty, it contains stat.comm as fallback
+    string cmdline = 3;
```

Comment:
Fields in oneofs must not have labels (required / optional / repeated).

Reply:
Yes, you would just define another message. I like that because the client knows what data it gets. The protobuf should define that clearly and not have internal implementation details again, which the client has to know about.

The client then knows that it has either a `comm` or the full `exe` and can decide without having to know that sometimes it gets one or the other which leaves a lot of ambiguity.

Reply:
Extra messages are just semantics and have as far as I am aware no overhead at all.

Reply:
Done, good arguments!